### ISSUE TITLE
Webpack 4: Can't import the named export for non EcmaScript module

### ISSUE THREAD

**Issue by katonada** [wopian/kitsu#155]
After switching one project to webpack4 I got error messages from the compiler (kitsu / kitsu-core). I tried to fix problems myself but on every new instalation the problem will remain. Thanx
```
Failed to compile with 13 errors                                                                        13:32:11

 error  in ./node_modules/kitsu/lib/index.mjs

102:122-131 Can't import the named export 'serialise' from non EcmaScript module (only default export is available)

 error  in ./node_modules/kitsu/lib/index.mjs

110:69-74 Can't import the named export 'error' from non EcmaScript module (only default export is available)

 error  in ./node_modules/kitsu/lib/index.mjs

131:132-141 Can't import the named export 'serialise' from non EcmaScript module (only default export is available)

 error  in ./node_modules/kitsu/lib/index.mjs

142:69-74 Can't import the named export 'error' from non EcmaScript module (only default export is available)

 error  in ./node_modules/kitsu/lib/index.mjs

172:68-73 Can't import the named export 'error' from non EcmaScript module (only default export is available)

 error  in ./node_modules/kitsu/lib/index.mjs

21:12-17 Can't import the named export 'camel' from non EcmaScript module (only default export is available)

 error  in ./node_modules/kitsu/lib/index.mjs

23:41-46 Can't import the named export 'snake' from non EcmaScript module (only default export is available)

 error  in ./node_modules/kitsu/lib/index.mjs

23:49-54 Can't import the named export 'kebab' from non EcmaScript module (only default export is available)

 error  in ./node_modules/kitsu/lib/index.mjs

45:43-48 Can't import the named export 'query' from non EcmaScript module (only default export is available)

 error  in ./node_modules/kitsu/lib/index.mjs

50:78-89 Can't import the named export 'deserialise' from non EcmaScript module (only default export is available)

 error  in ./node_modules/kitsu/lib/index.mjs

52:69-74 Can't import the named export 'error' from non EcmaScript module (only default export is available)

 error  in ./node_modules/kitsu/lib/index.mjs

73:59-68 Can't import the named export 'serialise' from non EcmaScript module (only default export is available)

 error  in ./node_modules/kitsu/lib/index.mjs

81:69-74 Can't import the named export 'error' from non EcmaScript module (only default export is available)
```

**Comment by wopian** [comment]
Seems Node and Webpack are incompatible when targeting ESM output. I've temporarily resolved this with `5.1.0` by removing support for ESM in its entirety.

When I have time, this will be properly fixed and `kitsu` will output ESM again.

https://github.com/wopian/kitsu/blob/3da9a540a0ff34cfb6d510db9a65a3b399debeaf/packages/kitsu/rollup.config.js#L68

**Comment by wopian** [comment]
Fixed in `6.1.0`
